packages/calypso-e2e/src/browser-manager.ts

```diff
@@ -116,3 +116,42 @@ export async function clearAuthenticationState( page: Page ): Promise< void > {
 	// Previous steps navigated page away from target page. Return page to the original URL.
 	await page.goto( currentURL );
 }
+
+/**
+ * Sets the store cookie, used for simulating payment processing.
+ *
+ * Optinally, set the `currency` parameter to specify the currency to be used in the checkout.
+ *
+ * @param {Page} page Object representing a page launched by Playwright.
+ * @param {string} currency ISO 4217-compliant currency code.
+ */
+export async function setStoreCookie(
```

Comment:
Instead of having multiple methods to set different cookies, enable additional cookies using flags.